src/amo/pages/UserProfile/index.js

```diff
@@ -416,6 +419,18 @@ export class UserProfileBase extends React.Component<InternalProps> {
                 {i18n.gettext('Edit profile')}
               </Button>
             ) : null}
+
+            {canAdminUser && user ? (
```

Comment:
nit: since you check for the user object when setting `canAdminUser` (https://github.com/mozilla/addons-frontend/pull/6851/files#diff-8ac5e13d9f69883f4468abd7d2787b98R493), is this needed again?

Reply:
Unfortunately Flow doesn't seem to be able to keep track of that, so while we know that we don't really need this check, Flow seems to require it.

Reply:
ahh..makes sense then. thanks!